client/src/main/java/sigblockchain/projectred/client/Block.java

```diff
@@ -0,0 +1,92 @@
+package sigblockchain.projectred.client;
+
+import java.math.BigInteger;
+
+public class Block {
+    private final int Version;          // uint16
+    private final BigInteger Height;          // uint64
+    private final long Timestamp;
+    private final byte[] PreviousHash;
+    private final byte[] MerkleRootHash;
+    private final int DataLen;          // uint16
+    private final byte[][] Data;        // [][]byte
+
+    /**
+     * @param version        Version of the blockchain the block was created on
```

Comment:
I added the Javadoc comment. In general it is good practice to have those comments on class and functions that will be widely used throughout the code base.

Reply:
Fixed!